services/docker-compose.yml

```diff
@@ -228,7 +228,7 @@ services:
         - traefik.http.services.${SWARM_STACK_NAME}_webserver.loadbalancer.sticky.cookie.secure=true
         - traefik.http.middlewares.${SWARM_STACK_NAME}_webserver_retry.retry.attempts=2
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver.service=${SWARM_STACK_NAME}_webserver
-        - traefik.http.routers.${SWARM_STACK_NAME}_webserver.rule=hostregexp(`{host:.+}`) && (Path(`/`, `/v0`,`/socket.io/`,`/static-frontend-data.json`, `/study/{study_uuid:\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b}`, `/view`, `/#/view`, `/#/error`) ||  PathPrefix(`/v0/`))
+        - traefik.http.routers.${SWARM_STACK_NAME}_webserver.rule=hostregexp(`{host:.+}`) && (Path(`/`, `/v0`,`/socket.io/`,`/static-frontend-data.json`, `/study/{study_uuid:\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b}`, `/view`, `/#/view`, `/#/error`) ||  PathPrefix(`/v0/`)  ||  PathPrefix(`/dev/`))
```

Comment:
I don't understand why you have /v0 in the first path and then also in PathPrefix. It should be from my understanding only in Pathprefix (so everything starting by PathPrefix V0 will be used.) I know this is not part of your PR but I think it could be corrected.

Also, I think you could have something like PathPrefix(`/v0/`,`/dev/` )

We agree that what you want here is that everything which is in the form DOMAIN/dev/* goes threw, right?

Reply:
@Surfict My interpretation, based on https://doc.traefik.io/traefik/routing/routers/#rule, is that
```
PathPrefix(`/v0/`)  ||  PathPrefix(`/dev/`)
```
is the same as your suggestion
```
PathPrefix(`/v0/`, `/dev/`)
```
but I think the latter is actually better, so I will change it. 

@Surfict my main concern is whether this overrides or gets overridden by the docker-compose you have in ops ... !!!
@sanderegg could you please give some feedback on this as well?

Reply:
Alright so, ```Path``` means: allow this exact path to be redirected. Anything else will not be
```PathPrefix``` means anything that starts with this pathprefix will be redirected.
Therefore if you put ```PathPrefix(`/v0`)``` then ```/v0```, ```/v0/projects``` will be redirected, but also ```/v0somepath```. And the latter we do not want.

Reply:
I do agree that ```PathPrefix(`/v0/`, `/dev/`)``` is a good start.
Now the real question is to know whether the ```/dev/``` path should be available publicly or not? If not then it should part of another router with auth.

Reply:
@Surfict @sanderegg so at the end ``PathPrefix(`/v0/`, `/dev/`)`` was added in ``docker-compose.local.yml`` (i.e. for local development) so it should NOT collide with compose-spec file added in operations.